src/actions/performer.ts

```diff
@@ -16,31 +11,31 @@ export async function _performAction(action: IAction): Promise<any> {
       { action }
     );
     switch (action.actionType) {
-      case ActionType.Command: {
+      case "COMMAND": {
```

Comment:
Because we removed the enums we now have "magic" string values.
We should define/declare all of these **once** in some **constants** helper file
to enable easy future refactoring/changes.